security/signing.py

```diff
@@ -48,6 +51,76 @@ def resolve_private_key(self, key_id: str):
         return private_key
 
 
+class BlenderCertificationResolver:
+    def create_csr(self, pvt_key):
+        csr = (
+            x509.CertificateSigningRequestBuilder()
+            .subject_name(
+                x509.Name(
+                    [
+                        x509.NameAttribute(NameOID.COUNTRY_NAME, "FI"),
+                        x509.NameAttribute(NameOID.STATE_OR_PROVINCE_NAME, "Pirkanmaa"),
+                        x509.NameAttribute(NameOID.LOCALITY_NAME, "Tampere"),
+                        x509.NameAttribute(NameOID.ORGANIZATION_NAME, "TII"),
+                        x509.NameAttribute(NameOID.COMMON_NAME, "blender.api"),
+                    ]
+                )
+            )
+            .add_extension(
+                x509.SubjectAlternativeName(
+                    [
+                        # Describe what sites we want this certificate for.
+                        x509.DNSName("http://localhost:8000/"),
+                    ]
+                ),
+                critical=False,
+                # Sign the CSR with private key.
+            )
+            .sign(pvt_key, hashes.SHA256())
+        )
+        return csr
+
+    def create_ca(self, pvt_key, csr):
```

Comment:
I guess, this is not required as blender pod will get the signed certificated as readonly mount accessible via env

Reply:
I think this is probably meant for testing purposes, so that we can create test certificates and leave the actual certs alone.

Reply:
Looks ok for unit test. Unit test Cert handling part can be decoupled from signing module.

Reply:
Yeah I will decouple the Cert creation